src/result_transformer.js

```diff
@@ -14,25 +17,32 @@ const transformDataToTable = (analyticsResult, options) => {
 const transformDataToTimeSeries = (analyticsResult, options) => {
   const groupBys = options.data.groupBy;
   const results = [];
+
+  const interval = options.data.interval;
+  const fromDate = new Date(options.data.start).getTime();
+  const toDate = new Date(options.data.end).getTime();
   if (groupBys.length > 0) {
     let groupings = {};
     _.map(analyticsResult.rows, row => {
       const metricLabel = row[1];
       if (!groupings[metricLabel]) {
         groupings[metricLabel] = [];
       }
-      groupings[metricLabel].push([row[2], row[0]]); // value, timestamp
+      const value = row[2];
+      const timestamp = row[0];
+      groupings[metricLabel].push([value, timestamp]);
     });
-    Object.keys(groupings).map(key => {
-      const datapoints = groupings[key];
+    for (let key of Object.keys(groupings)) {
+      const datapoints = groupings[key]; //padTimeSeriesAndSortByDate(groupings[key], fromDate, toDate, interval);
```

Comment:
let's remove this comment :)

Reply:
https://github.com/bitmovin/analytics-grafana-datasource/pull/21/commits/36c4ba49b6f00dc12190efb3a30544a8f4a00109